solutions/devsprint-cecilia-saraiva-1/GitHubApp/Screens/List/RepositoryCell/RepositoryCellView.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  RepositoryCellView.swift
+//  GitHubApp
+//
+//  Created by Pierre Youness on 26/07/2022.
+//
+
+import UIKit
+
+struct RepositoryCellVM {
```

Comment:
[suggestion] better to use full name.
```suggestion
struct RepositoryCellViewModel {
```
And it's nice you name view model, but it has no logic at this point.
Also maybe you can add in its own file.